CONTRIBUTING.md

```diff
@@ -177,24 +177,43 @@ exposed.
 
 ## Pull Requests
 
-The master development branch is `master`.
-Branch names should be prefixed with the author, eg. `name/feature-x`.
+If you have write access to the ibc-rs repo, you can directly branch off of `master`.
+This makes it easier for project maintainers to directly make changes to your
+branch should the need arise.
 
-Pull requests are made against `master`
-and are squash merged into master.
+Branch names should be prefixed with the author's name followed by a short description
+of the feature, eg. `name/feature-x`.
 
-PRs must:
+Pull requests are made against `master` and are squash-merged into master.
 
-- make reference to an issue outlining the context.
-- update any relevant documentation and include tests.
+PRs must:
+- make reference to an issue outlining the context
+- update any relevant documentation and include tests
 - add a corresponding entry in the `.changelog` directory using `unclog`,
   see the section above for more details.
 
-Pull requests should aim to be small and self contained to facilitate quick
+Pull requests should aim to be small and self-contained to facilitate quick
 review and merging. Larger change sets should be broken up across multiple PRs.
 Commits should be concise but informative, and moderately clean. Commits will be squashed into a
 single commit for the PR with all the commit messages.
 
+In order to help facilitate the PR review process, tag *one* person as the
```

Comment:
Should we update our PR template to reflect this (i.e. maybe add a checklist point to tag a reviewer)?

Reply:
Done 👍 
[e97e620](https://github.com/informalsystems/ibc-rs/pull/2438/commits/e97e620ece80f2f1bdbc7436053c88d775971a03)